Maquetado/scss/grid.scss

```diff
@@ -1,12 +1,28 @@
 .tech-container-grid {
   display: grid;
-  grid-template-columns: 1fr 1fr 1fr;
-  grid-template-rows: auto;
-  grid-template-areas:
-    "tech-a tech-b tech-c"
-    "tech-a tech-d tech-e";
-  margin-bottom: 70px;
-  padding: 0 300px;
+
+  @media screen and (min-width: 1024px) {
```

Comment:
don't use 2 media queries, 1 for big screens and one for small. use regular properties and override them for small screens using a media query